pyseir/inference/model_fitter.py

```diff
@@ -0,0 +1,478 @@
+        systematic errors of cases by this number. Thus higher numbers reduce
+        the influence of hospitalization data on the fit.
+    """
+    def __init__(self,
+                 fips,
+                 ref_date=datetime(year=2020, month=1, day=1),
+                 min_deaths=2,
+                 n_years=1,
+                 cases_to_deaths_err_factor=1,
+                 hospital_to_deaths_err_factor=1):
+
+        self.fips = fips
+        self.ref_date = ref_date
+        self.min_deaths = min_deaths
+        self.t_list = np.linspace(0, int(365 * n_years), int(365 * n_years) + 1)
+        self.cases_to_deaths_err_factor = cases_to_deaths_err_factor
+        self.hospital_to_deaths_err_factor = hospital_to_deaths_err_factor
+        self.t0_guess = 60
+
+        if len(fips) == 2: # State FIPS are 2 digits
+            self.agg_level = AggregationLevel.STATE
+            self.state_obj = us.states.lookup(self.fips)
+            self.state = self.state_obj.name
+            self.geo_metadata = load_data.load_county_metadata_by_state(self.state.title()).loc[self.state.title()].to_dict()
+            self.times, self.observed_new_cases, self.observed_new_deaths = \
+                load_data.load_new_case_data_by_state(self.state, self.ref_date)
+            self.hospital_times, self.hospitalizations, self.hospitalization_data_type = \
+                load_data.load_hospitalization_data_by_state(self.state_obj.abbr, t0=self.ref_date)
+            self.display_name = self.state
+        else:
+            self.agg_level = AggregationLevel.COUNTY
+            self.geo_metadata = load_data.load_county_metadata().set_index('fips').loc[fips].to_dict()
+            self.state = self.geo_metadata['state']
+            self.county = self.geo_metadata['county']
+            self.display_name = self.county + ', ' + self.state
+            # TODO Swap for new data source.
+            self.times, self.observed_new_cases, self.observed_new_deaths = \
+                load_data.load_new_case_data_by_fips(self.fips, t0=self.ref_date)
+            self.hospital_times, self.hospitalizations, self.hospitalization_data_type = \
+                load_data.load_hospitalization_data(self.fips, t0=self.ref_date)
+
+        self.cases_stdev, self.hosp_stdev, self.deaths_stdev = self.calculate_observation_errors()
+
+        self.fit_params = dict(
+            R0=3.4, limit_R0=[1, 8], error_R0=1.,
+            log10_I_initial=2, limit_log10_I_initial=[0, 5], error_log10_I_initial=.5,
+            t0=60, limit_t0=[0, 70], error_t0=1,
+            eps=.3, limit_eps=[0, 2], error_eps=.2,
+            t_break=20, limit_t_break=[0, 100], error_t_break=2,
+            test_fraction=.05, limit_test_fraction=[0.001, 1], error_test_fraction=.01,
+            hosp_fraction=1, limit_hosp_fraction=[0.001, 1], error_hosp_fraction=.1,
+            fix_hosp_fraction=self.hospitalizations is None, # Let's not fit this to start...
+            errordef=.5
+        )
```

Comment:
Better put these into a config file.

Reply:
I think they better here actually for two reasons: 
1. Some are programatic, like `fix_hosp_fraction`
2. These are physical constraints, and hence are not expected to change.